frontend/src/locales/fr.json

```diff
@@ -298,7 +298,7 @@
     "prompt": {
       "promptEntityDelete": {
         "title": "Confirmation de suppression",
-        "warningText": "Veux-tu vraiment le supprimer ?"
+        "warningText": "Veux-tu vraiment le supprimer? | Veux-tu vraiment supprimer \"{entity}\"?"
```

Comment:
The space in front of the question mark is normal in French, you should probably restore that